test/test-e2e.sh

```diff
@@ -22,9 +22,10 @@ setup_helm() {
   if ! [[ $(${HELM} version |grep Version |awk -F'Version:' '{print $2}' |awk -F',' '{print $1}') == "\"v${HELM_VERSION}\"" ]] ; then
       echo "helm not found or bad version, downloading binary"
       mkdir -p _output
-      wget https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz
-      tar -zxvf helm-v${HELM_VERSION}-linux-amd64.tar.gz
-      mv linux-amd64/helm _output/helm
+      curl -Lo /tmp/helm.tar.gz "https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz"
+      tar -xzf /tmp/helm.tar.gz -C /tmp
+      rm -f /tmp/helm.tar.gz
+      mv -f /tmp/linux-amd64/helm _output/helm
       chmod +x _output/helm
       HELM=_output/helm
```

Comment:
This could probably be simplified by doing something like:

```suggestion
      wget -qO- "https://get.helm.sh/helm-v${HELM_VERSION}-linux-amd64.tar.gz" | tar xvz -C _output/helm
      chmod +x _output/helm/linux-amd64/helm
      HELM=_output/helm/linux-amd64/helm
```

Reply:
@dgrisonnet I've copied the pattern used by other blocks in the script which wasn't used for Helm for some reason and resulted in unnecessary log bloat.

This is a release branch so any changes here will get lost for the next release, but I do have https://github.com/kubernetes-sigs/metrics-server/pull/1147 open if you'd like to add these comments there?

Reply:
Sure :+1:, sorry I didn't looked at whether this was a release branch or not